subtests/docker_cli/run_sigproxy/run_sigproxy.py

```diff
@@ -98,7 +112,7 @@ def initialize(self):
             self._init_container_attached(name)
         else:
             self._init_container_normal(name)
-        time.sleep(self.config['wait_start'])
+        self._wait_for_ready()
```

Comment:
Sorry, I wasn't clear: I meant, rewrite _wait_for_ready() so it uses the much nicer wait_for_output()?